src/frontend-samples/clash-review-sample/sampleSpec.ts

```diff
@@ -12,17 +12,18 @@ export function getClashReviewSpec(): SampleSpec {
     label: "Clash Review",
     image: "clashreview.png",
     readme: async () => import("!!raw-loader!./readme.md"),
+    iTwinViewerReady: true,
```

Comment:
There is a broken link in README.md 
- "Provide code examples" 
- "Clash Detection result sample data"
     - Either remove link or add ClashDetectionJsonData.ts to files

Reply:
fixed